tests/baselines/reference/aliasOnMergedModuleInterface.errors.txt

```diff
@@ -8,7 +8,7 @@ tests/cases/compiler/aliasOnMergedModuleInterface_1.ts(5,16): error TS2304: Cann
     z.bar("hello"); // This should be ok
     var x: foo.A = foo.bar("hello"); // foo.A should be ok but foo.bar should be error
                    ~~~
-!!! error TS2304: Cannot find name 'foo'.
+!!! error TS2691: Cannot find name 'foo'. A type exists with this name, but no value.

```

Comment:
Ah, ok. I was able to revert this. From my testing, I only ever needed to call resolveAlias once to see that its root was a namespace, even if the namespace was passed through multiple files.